update/ansible/playbook/tasks/update.yml

```diff
@@ -108,9 +109,15 @@
       when: is_docker and not is_supervisor_running.stat.exists
       shell: supervisord -c /etc/supervisord.conf &
 
+    - name: Wait until postgres socket is present before continuing
+      wait_for:
+        path: /var/run/postgresql/.s.PGSQL.5432
```

Comment:
in cases above we use port, but here we use socket. can we use the same everywhere. Asking, because we are working on support of external PostgreSQL.